### DISCUSSION TITLE
Issue with multiple Livewire components (each having 1 checkbox)

### DISCUSSION THREAD

**Discussion by burfbari** [livewire/livewire#5740] [1. Help]
> When a value attribute is present on the <input> element when it is checked, the Livewire property should update to that value, not a boolean like it currently does.

as I have got multiple Livewire components (each having 1 checkbox)
`<livewire:components.toggle-admin :userID="$user->userID" :wire:key="'user-'.$user->userID">`

the `id` seems to be getting passed fine
```
public $userID;
```

but value seems to be messed up:
```
<div x-data="{checked : '{{ $checked }}'}">
{{ $userID }}
<input
        class="hidden appearance-none"
        type="checkbox"
        id="check"
        value="{{ $userID }}"
        wire:model="checked"
        x-model="checked"
        @click="$wire.toggleAdminRights('{{ $userID }}')"
    >
```

The `userID` renders fine but when setting this as `value`, there seems to an issue.

![livewire-checkboxes](https://github.com/livewire/livewire/assets/52349056/e3ae457b-4f57-4316-96d2-c0f3bca77b31)

If `wire:model="checked"` is removed, the values start populating properly.

Any ideas?

**Comment by lrljoe** [comment]
Entangle your AlpineJS approach rather than writing a value.
It'll work fine then.

I'd suggest not having a whole component for just displaying a checkbox though.

You'd be better off with extracting the state for each user, and then calling a function in the parent component to update the state.

Otherwise you are probably making many calls to a db needlessly.

V3 does allow for smoother use of subcomponents, only a month away
